jenkins/tippecanoe_tile_join_segsAllConus_Jenkinsfile

```diff
@@ -45,7 +48,8 @@ pipeline {
         } 
       }
       steps {
-        sh 'tile-join --force -pk --output-to-directory tiles -c stream_temp_2015_10_25.csv tilesSegsAllConus'
+	    date=`cat date.txt`
```

Comment:
Do the environment variables not propagate from step to step?

Reply:
it's not for some reason, about to ask ivan for help! i don't get it.

Reply:
still fails here after i added this.

Reply:
guess the block was poorly formatted, i re-added it after asking ivan for help and now it works.

Reply:
https://jenkins.wma.chs.usgs.gov/job/MakerSpace/view/Water%20Temp/job/tippecanoe_tile_join_segsAllConus/28/console

Reply:
wonder if it's because this is inside the tippecanoe container and not just inside the makerspace node like the previous block